keyboards/c64/C64.h

```diff
@@ -0,0 +1,51 @@
+#ifndef C64_H
+#define C64_H
```

Comment:
```suggestion
```

Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Reply:
This file looks like it's an old version that should be removed.